### PR TITLE
[Bug 1119374] Fix up question topic charts

### PR DESCRIPTION
**Pull request by mythmon**
* Put the y-axis on the same line.
* Remove topics with no data.
* Fix the hover tool tips.

Since I re-indented a file, this is probably best viewed with [`?w=1`](2355/files?w=1).

r?

### CODE REVIEW

File: kitsune/questions/views.py [hunk 1]
```diff
@@ -1665,9 +1665,15 @@ def metrics(request, locale_code=None):
         start = date.today() - timedelta(days=30)
         end = date.today()
 
+    graph_data = stats_topic_data(bucket_days, start, end, locale_code, product)
+
+    for group in graph_data:
+        for name, count in group.items():
+            if count == 0:
+                del group[name]
```
Comment (rlr) [inline kitsune/questions/views.py]: missing `;`?

Comment (mythmon) [inline kitsune/questions/views.py]: Python?

Comment (rlr) [inline kitsune/questions/views.py]: rofl #friday

**Comment by rlr** [comment]
r+

**Pull request merged**